module/VuFind/src/VuFind/Search/Solr/Params.php

```diff
@@ -482,11 +487,14 @@ protected function normalizeSort($sort)
                     $order ?: $table[$field]['order']
                 );
             } else {
-                $normalized[] = sprintf(
+                $sortValue = sprintf(
                     '%s %s',
                     $field,
                     $order ?: 'asc'
                 );
+                if (!in_array($sortValue, $normalized)) {
```

Comment:
This won't prevent you from having a weird sort order like "id desc,id asc". Maybe the thing to do is to initialize a `$fields` array at the same time that we initialize `$normalized`, and then change this logic to:

```
if (!in_array($field, $fields)) {
    $normalized[] = $sortValue;
    $fields[] = $field;
}
```

This will ensure that no field appears in the sort list more than once -- it helps with tie-breakers but also with problems in the core sort configuration.

I believe this also means that we can now support complex, multi-part tie-breakers, like "title desc,id asc" if we want to. It might be worth adding a couple more cases to the unit test to cover these additional scenarios.